Dataset.py

```diff
@@ -267,6 +267,8 @@ def get_seq_order_for_epoch(self, epoch, num_seqs, get_seq_len=None):
     seq_index = list(range(num_seqs)); """ :type: list[int]. the real seq idx after sorting """
     if self.seq_ordering == 'default':
       pass  # Keep order as-is.
+    elif self.seq_ordering == 'reverse':
+      seq_index = list(reversed(seq_index))
```

Comment:
Just because this is possible with the current CombinedDataset. I don't actually need this ;)